src/eject/requiredDependencies.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * A list of dependencies which are installed into enclave, but will be necessary
+ * for a project if enclave is ejected.
+ * @type {string[]}
+ */
+var requiredDependencies = [
```

Comment:
I think this will be easier to manage if this file read from the `package.json` to populate the array. As we develop enclave, we don't want to manually maintain this.